### PR TITLE
Setup changes

### PR DESCRIPTION
**Pull request by MichaelClerx**
See #30 

Has a pinned version of sympy due to #31 

Also note this is a merge into cellml-parsing, so we can keep that as the branch for fccodegen :D

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,7 @@ language: python
 python:
     - "3.6"
 install:
-    - pip install -r requirements/test.txt
-    - pip install -e .
+    - pip install -e .[test]
```
Comment (jonc125) [inline .travis.yml]: Do the tests still pass without the `-e`? They should (and we should remove it).

Comment (MichaelClerx) [reply .travis.yml]: They didn't! But fixed that now :-)

Comment (jonc125) [reply .travis.yml]: And this is why Travis shouldn't use `-e` :)

**Comment by MichaelClerx** [comment]
Any comments @tamuri ? Otherwise I'll just merge

**Comment by tamuri** [comment]
Happy for you to merge.

**Pull request merged**